### PR TITLE
FASTPROCESS TGUI

### PR DESCRIPTION
**Pull request by Iamgoofball**
:joy:

### CODE REVIEW

File: code/controllers/subsystem/tgui.dm [hunk 1]
```diff
@@ -27,3 +32,4 @@ var/datum/subsystem/tgui/SStgui
 			ui.process()
 			continue
 		processing_uis.Remove(ui)
+		CHECK_TICK
```
Comment (MrStonedOne) [inline code/controllers/subsystem/tgui.dm]: This will make it sleep, fucking shit up, i'll pr it to your branch the proper way to do this

Comment (Iamgoofball) [inline code/controllers/subsystem/tgui.dm]:
~~~

**Comment by GunHog** [comment]
If MSO made the code...why not have him create the PR?

**Comment by optimumtact** [comment]
@GunHog because goof started this work?

**Comment by MrStonedOne** [comment]
So i'm getting reports that the fast updates are crashing some clients. we might have to shelve this for now.

**Pull request closed without merging**